test/extensions/filters/udp/dns_filter/dns_filter_test.cc

```diff
@@ -119,12 +119,6 @@ stat_prefix: "my_prefix"
 server_config:
   inline_dns_table:
     external_retry_count: 3
-    known_suffixes:
```

Comment:
You should have some DEPRECATED_FEATURE_TEST to make sure the code still works with a config with this in it. I'm not sure what it would do other than potentially warn to stop setting it since it doesn't do anything? I guess that's not even useful since it would print a deprecated field warning, but I would still make sure it's covered.

Reply:
ack.  I'll look into adding one.  Thanks!